src/perfrecord.cpp

```diff
@@ -62,72 +63,27 @@ PerfRecord::~PerfRecord()
     }
 }
 
-static QStringList sudoOptions(const QString& sudoBinary)
```

Comment:
the old code needs to stay for people like me who don't pollute their `/usr` by installing stuff in there. similarly the appimage won't be able to use kauth either. afaik kauth is only an option for distro packages.